test/run-test.ps1

```diff
@@ -50,12 +50,10 @@ Try {
         $Architecture = "amd64"
     }
 
-    $TestFilter = "Architecture=$Architecture"
-    if (![string]::IsNullOrWhiteSpace($Filter)) {
-        $TestFilter += "&Version~$filter"
-    }
+    $env:IMAGE_ARCH_FILTER = $Architecture
```

Comment:
Should these environment variable be deleted towards the end of test in a finally block - `Remove-Item Env:\IMAGE_ARCH_FILTER -ErrorAction Ignore`

Reply:
I don't think it is necessary - these env vars are only set for the current PS session and not machine wide.